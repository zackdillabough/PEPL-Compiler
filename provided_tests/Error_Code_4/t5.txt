POLY T(X, Y) = X + Y;
POLY S(R, Y, U) = R^2 + Y^2 + U;
POLY U(x, y) = x + 5555;
START
INPUT A;
INPUT B;
INPUT C;
T(
A, C
);
S(A, 
B, C);






U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U(U

(U(U(U(U(U(U(U(U(U(U(U(U(

U(U(U(U(U(U(U(U(U(B))))))))))))))))))))



))))))))))))))))))))))))))))));

1 2 3 4 5
