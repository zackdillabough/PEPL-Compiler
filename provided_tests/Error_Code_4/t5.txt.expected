Error Code 4: 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 19 21 21 21 21 21 21 21 21 21 21 21 21 23 23 23 23 23 23 23 23 23 